utils/src/main/kotlin/OrtProxySelector.kt

```diff
@@ -171,6 +177,12 @@ class OrtProxySelector(private val fallback: ProxySelector? = null) : ProxySelec
     override fun select(uri: URI?): List<Proxy> {
         requireNotNull(uri)
 
+        if (proxyIncludes.isNotEmpty()) {
```

Comment:
How about `if (proxyIncludes.isNotEmpty()) return NO_PROXY_LIST` to get rid of this nesting?

Reply:
That code would be wrong, because it wouldn't handle excludes (or proxies at all). The logic is: If `proxyIncludes` are empty, includes are not restricted, and every host is included.

Reply:
Sorry, I mean `isEmpty()` - anyhow maybe it's also wrong. Please ignore if so.